src/objects/Hierarchy_Settings__c.object

```diff
@@ -95,29 +95,29 @@
         <defaultValue>false</defaultValue>
-        <description>Determines if the Affiliation automatically created by creating a Program Enrollment will inherit the Program Enrollment&apos;s Start Date.</description>
+        <description>Indicates whether to set the End Date of the Affiliation created with a Program Enrollment. If enabled, set the End Date to the Program Enrollment&apos;s End Date. If disabled, don't specify the Affiliation&apos;s End Date.</description>
         <externalId>false</externalId>
-        <inlineHelpText>Determines if the Affiliation automatically created by creating a Program Enrollment will inherit the Program Enrollment&apos;s Start Date.</inlineHelpText>
-        <label>Affl End Date on Prog Enroll Creation</label>
+        <inlineHelpText>Indicates whether to set the End Date of the Affiliation created with a Program Enrollment. If enabled, set the End Date to the Program Enrollment&apos;s End Date. If disabled, don't specify the Affiliation&apos;s End Date.</inlineHelpText>
+        <label>Affil End Date Copy from Prog Enroll</label>
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
     <fields>
         <fullName>Affl_ProgEnroll_Copy_Start_Date__c</fullName>
         <defaultValue>false</defaultValue>
-        <description>Determines if the Affiliation automatically created by creating a Program Enrollment will inherit the Program Enrollment&apos;s Start Date.</description>
+        <description>Indicates whether to set the Start Date of the Affiliation created with a Program Enrollment. If enabled, set the Start Date to the Program Enrollment&apos;s Start Date. If disabled, don't specify the Affiliation&apos;s Start Date.</description>
         <externalId>false</externalId>
-        <inlineHelpText>Determines if the Affiliation automatically created by creating a Program Enrollment will inherit the Program Enrollment&apos;s Start Date.</inlineHelpText>
-        <label>Affl Start Date on Prog Enroll Creation</label>
+        <inlineHelpText>Indicates whether to set the Start Date of the Affiliation created with a Program Enrollment. If enabled, set the Start Date to the Program Enrollment&apos;s Start Date. If disabled, don't specify the Affiliation&apos;s Start Date.</inlineHelpText>
+        <label>Affil Start Date Copy from Prog Enroll</label>
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
     <fields>
         <fullName>Affl_ProgEnroll_Del_Status__c</fullName>
-        <description>Status to change an Affilation to when the related Program Enrollment is deleted, if the option selected is to change its status instead of deleting it.</description>
+        <description>If the related setting, Affil Delete With Prog Enroll, is disabled, this setting specifies the value to set for the Affiliation's Status, when its related Program Enrollment is deleted.</description>
```

Comment:
We changed the label of the related setting and forgot (as in, I forgot) to update it here in this description: "If the related setting, Affil Delete When Delete Prog Enroll,"